share/spack/gitlab/cloud_pipelines/.gitlab-ci.yml

```diff
@@ -724,8 +704,8 @@ tutorial-build:
     SPACK_CI_STACK_NAME: ml-linux-x86_64-cpu
 
 ml-linux-x86_64-cpu-generate:
-  extends: [ ".generate-x86_64", .ml-linux-x86_64-cpu, ".tags-x86_64_v4" ]
-  image: ghcr.io/spack/linux-ubuntu22.04-x86_64_v2:v2024-01-29
+  extends: [ ".generate", .ml-linux-x86_64-cpu, ".tags-x86_64_v4" ]
+  image: ghcr.io/spack/linux-ubuntu22.04-x86_64_v2:nightly
```

Comment:
Ditto the arch-specific generate thing.  And we don't want a nightly image here.